tensorflow/buildspec.yml

```diff
@@ -20,6 +25,18 @@
       deep_learning_container:
         source: docker/build_artifacts/deep_learning_container.py
         target: deep_learning_container.py
+      sagemaker_package_name:
```

Comment:
Should add one line of `inference_context: &INFERENCE_CONTEXT`, between line 27 and line 28

Please refer https://github.com/aws/deep-learning-containers/blob/master/tensorflow/buildspec-tf1.yml#L28

Please take care of indentation.

Reply:
done.